kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/services/ProductOrderService.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.good.patterns.orders.ordering.services;
+
+import com.kodilla.good.patterns.orders.ordering.*;
+
+import java.time.LocalDate;
+
+public class ProductOrderService implements OrderService {
```

Comment:
I have problem with name ProductOrderService (interface OrderService is designed also to order products). In this case it could be difficult to find good name because implementation is rather simple, so I propose SimpleOrderService or DefaultOrderService or OrderServiceImpl.